catboost/libs/documents_importance/docs_importance.cpp

```diff
@@ -13,6 +13,7 @@
 #include <util/generic/ymath.h>
 #include <util/string/cast.h>
 #include <util/string/iterator.h>
+#include <catboost/libs/logging/logging.h>
```

Comment:
The order of includes is the following (see style guilde) - first the files in same directory, then the files in same folder and so on. Then files in library folder, then in util folder. So this include should be earlier.